docs/contributing/index.rst

```diff
@@ -30,6 +30,16 @@ You can use `make` to install all tsuru dependencies and run tests. It will also
 
     $ make
 
+Please ensure yourself that MongoDB and Redis are started before running the tests (if you are seeing some failing tests the most likely reason is that these services are not started).
```

Comment:
sure @fsouza! I will fix this is another PR (with other minor stuff that I found today browsing  the source code/documentation (typos and that kind of stuff) since this one got merged hehe, is that ok?